src/utils.ts

```diff
@@ -0,0 +1,28 @@
+export const addClass = (el: HTMLElement, className: string) => {
+  if (el.classList) {
+    el.classList.add(className);
+  } else {
+    if (el.className.indexOf(className) < 0) {
```

Comment:
If class is `flicking-bullet-aa` and add `flicking-bullet` class , would it be a problem?

Reply:
Yeah, that can be a problem. I'll fix that.